extras/n3lo_bench/MSHT/split.F

```diff
@@ -0,0 +1,828 @@
+* =====================================================================
```

Comment:
Also not sure we want to keep this file ... again, maybe gitignore + `curl;tar;`

Reply:
About this I agree only if the location is known to be stable. Otherwise it might be better to keep in the repo

Reply:
okay I can add the automatic download to `f2py.sh`